src/go/k8s/tests/e2e/produce-consume/01-redpanda-cluster.yaml

```diff
@@ -12,10 +12,10 @@ spec:
   resources:
     requests:
       cpu: 1
-      memory: 2Gi
+      memory: 100M
```

Comment:
Did you have any problem with 2Gi?

Reply:
yes, the test cluster ran out of resources. I think we don't need that much, do we?

Reply:
Great that you find this problem that early. With our e2e tests should be enough. In performance tests probably not.